Exercise_D_E/assets/style/index.css

```diff
@@ -2,3 +2,97 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+body {
+  padding: 0;
+  margin: 0;
+}
+
+header {
+  width: 100vw;
+  height: 60 px;
+  background-color: #ffffff;
+}
+
+header img {
+  height: 100%;
+  padding-top: 10px;
+  padding-left: 80px;
+}
+
+.hero {
+    background-image: url("../images/flowershop.jpg");
+    height: 70vh ;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.overlay {
+    width: 100%;
+    height: 100%;
+    background-color: #000000;
+    opacity: 0.5;
+    position: absolute;
+  }
+.heading {
+  z-index: 5;
+  font-family: sans-serif;
+  width: 400px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+.heading h1 {
+    color: white;
+    font-size: 40px;
+}
+.heading button {
+    color: white;
+    font-size: 20px;
+    height: 50px;
+    width: 130px;
+    border: 0;
+    background-color: rgb(64, 202, 64);
+    border-radius: 30px;
+}
+
+
+.icons {
+    background-color: #cee7b5;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    padding-top: 40px;
+    padding-bottom: 50px;
+}
+.icon {
+    width: 40%;
+    height: 140px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.icon img{
+    height: 80%;
+}
+
+.icon p{
+    width: 50%;
+    padding-left: 10%;
+}
+
+@media only screen and (max-width: 700px) {
+  .icon {
+    width: 70vw;
+    flex-direction: column;
+    height: 300px;
+  }
+  .icon img{
+    height: 120;
+  }
+  .icon p{
+    font-size: 22px;
+    width: 60vw;
+  }
+}
```

Comment:
You have recreated the website exactly like how it's supposed to be with fantastic use of flex and linear gradient. Excellent work!